Tests/FoundationInternationalizationTests/CalendarTests.swift

```diff
@@ -686,6 +686,46 @@ final class CalendarBridgingTests : XCTestCase {
 }
 #endif
 
+
+extension DateComponents {
+    init(field: Calendar.Component, value: Int) {
```

Comment:
We actually don't need this in the tests - they import as testable so they can just use the internal one in the package.

Reply:
Perhaps we should consider adding this as API, though - it's clearly useful.

Reply:
But yours from https://github.com/apple/swift-foundation/pull/322/files#diff-10a4fcffe1d9245a1a5371a1ce086a0a67726b8dfde2454781613fecea34469bR82 isn't merged yet so I can't use it yet!

Reply:
I'll replace this later when yours is ready.

Reply:
Maybe just copy/paste the implementation so the merge is obvious?

Reply:
Done!